testkit/javadsl/src/test/java/com/lightbend/lagom/javadsl/testkit/services/PublishEvent.java

```diff
@@ -3,11 +3,13 @@
  */
 package com.lightbend.lagom.javadsl.testkit.services;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class PublishEvent {
 
     private final int code;
 
-    public PublishEvent(int code){
+    public PublishEvent(@JsonProperty("code") int code){
```

Comment:
We are using `@JsonCreator` in some other places. Maybe change this to use it too?